### ISSUE TITLE
Triggered builds don't send status

### ISSUE THREAD

**Issue by turuluu** [rockwelln/buildbot-slack#1]
Hey and congrats for the new plugin - it's proven useful to me and my team.  

When a build is triggered via commits (e.g. `SingleBranchScheduler`) the reporter works fine. However, when it is triggered via a buildstep (`Trigger`) that uses a `Triggerable` Scheduler, the notification doesn't happen.

My first assumption is that this is due to that `HttpStatusPushBase` on buildbot's behalf is not running the `getBuildDetailsAndSendMessage` in the first place. If you're up to it and have a debugging environment setup up, it would be nice to validate this and propagate to buildbot if necessary.

**Comment by rockwelln** [comment]
Hey, thanks for reporting your issue.
I'll try to prepare a minimal setup to reproduce your problem in the coming days.

**Comment by rockwelln** [comment]
Hello again,
I tried a setup with a buildstep Trigger using:
``` python
...
steps.Trigger(schedulerNames=["b01"], waitForFinish=True)
...
fake_triggerable = schedulers.Triggerable(name="b01", builderNames=["builder_01"])
c['schedulers'] = [
    fake_triggerable,
]
```
and I got the 4 Slack notifications as expected (2 for the top builder and 2 for the sub-builder b01).
to let me go further, can you investigate how your setup is different from what I tested?

buildbot version: 2.4.1
buildbot-slack: 0.1.0 (I also tried with master's HEAD)

**Comment by turuluu** [comment]
I think the issue is that I'm not using codebase or other change related constructs for the triggered builds, but instead I'm manually pulling a collection of repos and using `alwaysUseLatest` where applicable. Thus the changes won't register, but instead buildbot creates a fake commit for those.

The Console View shows:
`Fake comment for revision: No change for this revision, please setup a changesource in Buildbot`

Also I've setup `waitForFinish=False`

**Comment by rockwelln** [comment]
indeed, I was able to reproduce your problem using `alwaysUseLatest` in my minimal setup.
as stated in [buildbot doc](http://docs.buildbot.net/latest/manual/configuration/buildsteps.html#triggering-schedulers):

> alwaysUseLatest
>
>    If True, then **no SourceStamps are given**, corresponding to using the latest revisions of the repositories specified in the Source steps. This is useful if the triggered builds use to a different source repository.

Working on a fix to handle that...

**Comment by rockwelln** [comment]
it should be fixed in the new version 0.2.1.
you may update with `pip3 install -U buildbot-slack==0.2.1`

**Comment by turuluu** [comment]
very good - did indeed fix the issue.
